connect/api/src/main/java/org/apache/kafka/connect/errors/package-info.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * Provides common exception classes for Connect.
```

Comment:
We should make it clear that these exception classes establish a two-way API, between connectors/tasks/etc. and the framework:
```suggestion
 * Provides common exception classes for Connect.
 * Some may be thrown by the Connect runtime if connectors, tasks, or other components perform invalid or failed operations,
 * and others may be thrown by connectors, tasks, or other components to signal failures to the Connect runtime.
```